common/src/main/java/com/redhat/cloud/notifications/models/WebhookProperties.java

```diff
@@ -1,51 +1,33 @@
 package com.redhat.cloud.notifications.models;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.databind.annotation.JsonNaming;
 import com.redhat.cloud.notifications.db.converters.HttpTypeConverter;
-import com.redhat.cloud.notifications.models.validation.ValidNonPrivateUrl;
 import jakarta.persistence.Column;
 import jakarta.persistence.Convert;
 import jakarta.persistence.Entity;
 import jakarta.persistence.Table;
 import jakarta.persistence.Transient;
 import jakarta.validation.Valid;
-import jakarta.validation.constraints.NotNull;
-import jakarta.validation.constraints.Size;
-
-import static com.fasterxml.jackson.databind.PropertyNamingStrategies.SnakeCaseStrategy;
 
 @Entity
 @Table(name = "endpoint_webhooks")
-@JsonNaming(SnakeCaseStrategy.class)
 public class WebhookProperties extends EndpointProperties implements SourcesSecretable {
 
-    @NotNull
-    @ValidNonPrivateUrl
```

Comment:
General comment: I'm not sure we should remove the `jakarta.validation.constraints` annotations or `@ValidNonPrivateUrl` from the entities. They are the guarantee that no entity will ever be persisted with an invalid field. We also need them in the DTOs to reject any invalid REST request so maybe it's simply fine to have them twice in the code?

Reply:
Okay, yes, that makes sense. I'm bringing back the annotations!